web/marketing/pages/free-offers.html

```diff
@@ -27,24 +26,45 @@
                     <div class="col">{{printf "%.10s" .CreatedAt}}</div>
                     <div class="col">{{printf "%.10s" .ExpiresAt}}</div>
                     <div class="col"><a href="#">{{if eq .Status 1}}<strong>Live &#183;</strong>{{else}}off{{end}}</a></div>
-                    <div class="col col-heading edit-offer">EDIT</div>
                 </div><hr>
             {{end}}
         {{end}}
-        <div class="row offer-heading ">
-            <p class="offer-type">CUSTOM&nbsp;OFFERS</p>
-        </div>
-        {{range .FreeCredits}}
-        <div class="row data-row">
-            <div class="col ml-3">{{.Name}}</div>
-            <div class="col">${{.InviteeCreditInCents}}</div> 
-            <div class="col">0</div>
-            <div class="col">{{.RedeemableCap}}</div>
-            <div class="col">{{printf "%.10s" .CreatedAt}}</div>
-            <div class="col">{{printf "%.10s" .ExpiresAt}}</div>
-            <div class="col"><a href="#">{{if eq .Status 1}}<strong>Live &#183;</strong>{{else}}off{{end}}</a></div>
-            <div class="col col-heading edit-offer">EDIT</div>
-        </div><hr>
+    <div class="row offer-heading ">
+        <p class="offer-type">current&nbsp;offer</p>
+    </div>
+    {{range .FreeCredits}}
+        {{if eq .Status 2}}
+            <div class="row data-row">
+                <div class="col ml-3">{{.Name}}</div>
+                <div class="col">${{.InviteeCreditInCents}}</div>
+                <div class="col">{{.NumRedeemed}}</div> 
+                <div class="col">{{.RedeemableCap}}</div>
+                <div class="col">{{printf "%.10s" .CreatedAt}}</div>
+                <div class="col">{{printf "%.10s" .ExpiresAt}}</div>
+                <div class="col stop-offer">
+                    <span data-toggle="modal" data-target=".stop-free-credit-modal">
+                        <strong class="stop-offer">Live &#183;</strong>
+                    </span>
+                </div>    
+            </div><hr>
         {{end}}
+    {{end}}
+    {{template "stopFreeCredit" .}}
+    <div class="row offer-heading ">
+        <p class="offer-type">other&nbsp;offers</p>
```

Comment:
`Other&nbsp;Offers`, write the header as if it were without styling. Title case is usual for titles. So the HTML contains title-case, but css transforms it as needed. This makes screen readers understand better what's happening.